CONTRIBUTING.md

```diff
@@ -76,6 +76,10 @@ Maintainers are:
 ### PR Checklist
 
 Before your PR can be merged it must meet the following criteria:
+* PR Desciption should be informative, explaining what is in it and a high level design in case of new feature
```

Comment:
```suggestion
* Informative PR Description alongside an overview of the changes, high-level design in case of new feature
```

If you don't choose to go forward with this, note there's a typo `Desciption`

Reply:
yeah noted